activesupport/lib/active_support/core_ext/date_and_time/calculations.rb

```diff
@@ -9,15 +9,26 @@ module Calculations
       :saturday  => 5,
       :sunday    => 6
     }
+    WEEKEND_DAYS = [ 6, 0 ]
 
     # Returns a new date/time representing yesterday.
     def yesterday
-      advance(:days => -1)
+      advance(days: -1)
+    end
+
+    # Returns a new date/time representing the previous day.
+    def prev_day
+      advance(days: -1)
     end
 
     # Returns a new date/time representing tomorrow.
     def tomorrow
-      advance(:days => 1)
+      advance(days: 1)
+    end
+
+    # Returns a new date/time representing the next day.
+    def next_day
```

Comment:
It could have been, and @georgeclaghorn initially implemented it as such, but I find that more confusing. Clarity > DRY.